tests/corrupt-repo-ref.js

```diff
@@ -47,7 +47,7 @@ function listObjectChecksumsRecurse(dir, allObjects) {
     e.close(null);
 } 
 
-let [,commit] = repo.resolve_rev(refToCorrupt, false);
+repo.resolve_rev(refToCorrupt, false);
```

Comment:
This is basically dead code now.

Reply:
Ah, I'd assumed it was being called for some side-effect.